pkgs/tools/typesetting/tex/pplatex/default.nix

```diff
@@ -0,0 +1,40 @@
+with 
+  import <nixpkgs> {}
+;
```

Comment:
Instead of these three lines, use:
```
{ cmake, stdenv, pkgconfig, pcre, texlive, fetchFromGitHub }:
```
And then add a line for you package in `pkgs/top-level/all-packages.nix` like:
```
pplatex = callPackage ../tools/typesetting/tex/pplatex {};
```
To test you package, run `nix-build /path/to/nixpkgs/clone -A pplatex`.